build.gradle.kts

```diff
@@ -40,7 +40,7 @@ dependencies {
     implementation("io.micrometer:micrometer-core:1.9.+")
     runtimeOnly("org.jetbrains.kotlinx:kotlinx-coroutines-core-jvm:1.5.0")
     runtimeOnly("it.unimi.dsi:fastutil:8.5.2")
-    runtimeOnly("com.google.guava:guava:31.1-jre")
+    runtimeOnly("com.google.guava:guava:32.1.2-jre")
```

Comment:
We should test if this works with the gradle plugin. We have an issue with another guava dependency that caused an incompatibility issue. If the build works, then it is fine.